pytest/empty-contract-rs/Cargo.toml

```diff
@@ -12,8 +12,6 @@ serde = { version = "1", features = ["derive"] }
 serde_json = "1"
 wee_alloc = "0.4.5"
```

Comment:
Also remove `serde` import above (can't comment on in with github UI)

```suggestion
```

Reply:
Awesome, thanks. Done in https://github.com/nearprotocol/nearcore/pull/2991/commits/5c07c21f0057a0cf9a1c63810fcc1006daa1f8ac